jwst/extract_1d/extract.py

```diff
@@ -3191,12 +3183,8 @@ def is_prism(input_model: DataModel) -> bool:
 
     prism_mode = False
 
-    if (
-            instrument == "MIRI"
-            and instrument_filter.find("P750L") >= 0
-            or instrument == "NIRSPEC"
-            and grating.find("PRISM") >= 0
-    ):
+    if (instrument == "MIRI" and instrument_filter.find("P750L") >= 0 or
+            instrument == "NIRSPEC" and grating.find("PRISM") >= 0):
```

Comment:
This looks like a bug to me. It is very possible that the real intent was:
```suggestion
    if ((instrument == "MIRI" and instrument_filter.find("P750L") >= 0) or
         (instrument == "NIRSPEC" and grating.find("PRISM") >= 0)):
```
but I am not familiar with this.

Reply:
Agree with @mcara on this.